hazelcast/src/test/java/com/hazelcast/mapreduce/MapReduceTest.java

```diff
@@ -44,7 +44,7 @@
 @Category(QuickTest.class)
 @TestProperties(maxThreadsCalculatorClass = MapReduceTestThreads.class)
 @SuppressWarnings("unused")
-public class MapReduceTest
+    public class MapReduceTest
```

Comment:
FYI: CheckStyle didn't complain because test files are not analyzed at the moment.